Sources/Build/BuildPlan/BuildPlan.swift

```diff
@@ -786,7 +788,7 @@ extension BuildPlan {
                 components: [
                     package.identity.description,
                     module.name,
-                    module.buildTriple.rawValue,
```

Comment:
Are there any remaining uses of `buildTriple` or can this property be removed now?

Reply:
Yeah, we have a few more uses, mostly tests though. I'm planning to follow-up with SourceKitLSPAPI PR and then a PR to remove the rest.